<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
/* Group Box */
QGroupBox {
    background-color: white; /* Light blue background color */
    border: none;
border-radius:80px
}

/* Line Edits */
QLineEdit {
    border: 2px solid #003366; /* Dark blue border */
    border-radius: 5px;
    padding: 5px;
    background-color: white;
}

/* Hover Effect for Line Edits */
QLineEdit:hover {
    border-color: #005f5f; /* Darker blue on hover */
}

/* Focus Effect for Line Edits */
QLineEdit:focus {
    border-color: #007f7f; /* Even darker blue when focused */
}

/* Placeholder Text Color */
QLineEdit::placeholder {
    color: #003366; /* Dark blue placeholder text */
}

/* Push Button */
QPushButton {
    border-radius: 20px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #003366, stop:1 #005f5f); /* Dark blue to darker blue gradient */
    color: white;
    padding: 10px 20px;
    border: none;
}

/* Hover State Style for Push Button */
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #005f5f, stop:1 #007f7f); /* Darker blue to even darker blue gradient on hover */
}

/* Pressed State Style for Push Button */
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #007f7f, stop:1 #009f9f); /* Even darker blue to very dark blue gradient when pressed */
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Push Button */
QPushButton {
    border-radius: 20px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #001F3F, stop:1 #004080); /* Changed gradient colors */
    color: white;
    padding: 10px 20px;
    border: none;
}

/* Hover State Style for Push Button */
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #004080, stop:1 #001F3F); /* Changed gradient colors */
}

/* Pressed State Style for Push Button */
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #001F3F, stop:1 #004080); /* Changed gradient colors */
}</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAge">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>age</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Your Account</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>811</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/backdrop.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
